k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml

```diff
@@ -12,9 +12,9 @@ spec:
     enable: true
     retry: true
   chart:
-    repository: https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts
+    repository: https://github.com/Azure/aad-pod-identity/tree/master/charts
     name: aad-pod-identity
-    version: 2.0.1
+    version: 1.6.2
```

Comment:
This should be 2.0.1 as far as I can see

Reply:
This might not work i feel.